docs/synapse/userguides/storm_ref_automation.rstorm

```diff
@@ -0,0 +1,377 @@
+
+Conversely, triggers **only** execute when the specified event occurs. The events used to fire triggers typically will only ever occur once - the node ``inet:fqdn=woot.com`` will only ever be added to a Cortex once; applying the tag ``#my.tag`` to a ``hash:md5`` node will only occur one time (barring deleting and later re-adding the node or tag). The only exception is the ``prop:set`` event, which will fire when the specified property is first set (often on node creation) as well as any time the property is modified (i.e., "set" again).
+
+The "one-time" nature of triggers may result in a small number of edge cases where the trigger does not perform the desired action. For example:
+
+- Triggers do not operate retroactively; they will not fire for nodes that already exist in a Cortex at the time the trigger is added. That is, if you write a new trigger to fire any time the tag ``#my.tag`` is applied to a ``hash:md5`` node, the trigger will have no effect on existing ``hash:md5`` nodes that already have the tag.
+- If a trigger depends on a resource (process, service, etc.) that is not available when it fires, the trigger will simply fail; it will not "try again".
+
+Finally, users should be aware that in some cases proper trigger execution may depend on the timing and order of events with respect to creating nodes, setting properties, creating nodes from secondary properties ("derivative nodes"), etc. The detailed technical aspects of Synapse write operations are beyond the scope of this discussion, but as always it is good practice to test triggers (or other automation) before putting them into practice to ensure they perform as intended.
+
+**Syntax:**
+
+Triggers are created, modified, viewed, enabled, disabled, and deleted using the Storm ``trigger.*``  commands. See the :ref:`storm-trigger` command in the :ref:`storm-ref-cmd` document for details.
+
+.. NOTE::
+  Triggers can be modified after they are created, but modification is limited to updating the Storm query executed by the trigger. If other aspects of a trigger need to be changed (such as the event that fires the trigger, or the form a trigger operates on), the trigger must be deleted and re-created.
+
+**Examples:**
+
+In the examples below, the command to create (add) the specified trigger is shown.
+
+For illustrative purposes, in the **first** example the newly created trigger is displayed by using the ``trigger.list`` command and then by lifting the associated ``syn:trigger`` runtime ("runt") node.
+
+
+- Add a trigger that fires when an ``inet:whois:email`` node is created. If the email address is associated with a privacy-protected registration service (e.g., the email address is tagged ``#whois.private``), then also tag the ``inet:whois:email`` node.
+
+
+.. storm-pre:: trigger.add node:add --form inet:whois:email --query { +{ -> inet:email +#whois.private } [ +#whois.private ] } 
```

Comment:
This should be a ``storm:cli`` directive since we're showing how to add and we can let it execute fine

Reply:
We should add `--name` argument here

Reply:
done